### PR TITLE
Add tests for tenantid processor

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <p.loffay@gmail.com>

### CODE REVIEW

File: processors/tenantidprocessor/testdata/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+receivers:
+  examplereceiver:
+
+processors:
+  hypertrace_tenantid:
+    tenantid_header_name: header-tenant
+    tenantid_attribute_key: attribute-tenant
```
Comment (pavolloffay) [inline processors/tenantidprocessor/testdata/config.yml]: the tenantid prefix seems redundant I would remove it.

Comment (jcchavezs) [reply processors/tenantidprocessor/testdata/config.yml]: The indeed feel redundant but I am curious about the need to be configurable at least for header_name cc @davexroth

Comment (pavolloffay) [reply processors/tenantidprocessor/testdata/config.yml]: the config is optional, if no provided it uses the default that the platform uses

Comment (davexroth) [reply processors/tenantidprocessor/testdata/config.yml]: agreed, the tenantid_ prefix is redundant, feel free to remove.

having the key configurable is useful, as we've configured it with different headers in different deployments.

**Comment by jcchavezs** [comment]
Nice work. Consider adding a case in the e2e test: https://github.com/hypertrace/hypertrace-collector/tree/main/_tests/e2e. It shouldn't be that hard to add this processor.

**Pull request merged**